app/src/main/java/co/condorlabs/emerald/components/radiogroup/EmeraldRadioButton.kt

```diff
@@ -0,0 +1,84 @@
+package co.condorlabs.emerald.components.radiogroup
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.RadioButton
+import androidx.compose.material.RadioButtonColors
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.unit.dp
+import co.condorlabs.emerald.components.button.EmeraldButtonStyle
+import co.condorlabs.emerald.components.text.EmeraldTextStyle
+import co.condorlabs.emerald.theme.EmeraldDimens
+
+@Composable
+fun EmeraldRadioButton(
```

Comment:
this fun is exclusive of Emerald, this should not be implemented outside, please try to make it internal

`internal fun`

i am not sure if this works with funs but please try it :)

Reply:
Done